config.go

```diff
@@ -273,6 +273,7 @@ type config struct {
 
 	UnsafeDisconnect   bool   `long:"unsafe-disconnect" description:"Allows the rpcserver to intentionally disconnect from peers with open channels. USED FOR TESTING ONLY."`
 	UnsafeReplay       bool   `long:"unsafe-replay" description:"Causes a link to replay the adds on its commitment txn after starting up, this enables testing of the sphinx replay logic."`
+	SafeMode           bool   `long:"safemode" description:"Start in safe mode that forbids broadcasting of all commitment transactions"`
```

Comment:
perhaps we should call this something more descriptive than safe mode, e.g. `ForbidForceClose`?

Reply:
+1, I think the term safe in lightning is quite a strong word 😅